src/Components/SnackbarMessage.jsx

```diff
@@ -26,8 +26,8 @@ const SnackBarMessage = ({ message, type, open }) => {
       autoHideDuration={6000}
```

Comment:
(sorry for comment down here.. gh won't let me make file comment...)

pls move useStyles to end, convert const arrow to function and move styles from inline to useStyles section.